src/views/documentation-view.js

```diff
@@ -37,28 +42,49 @@ class DocumentationView extends ViewMixin(ReduxMixin(PolymerElement)) {
         width: auto;
         max-width: 100%;
     }
+        
+    .version-selector {
+        padding-left: 20px;    
+    }
+        
+    .version-selector paper-dropdown-menu {
+        width: 60px;
+        margin-left: 20px;
+    }
 </style>
 
 <app-drawer-layout narrow="{{narrow}}">
     <app-drawer id="drawer" slot="drawer" swipe-open="[[narrow]]" opened="{{drawerOpened}}">
         <div class="panel-menus">
+            <div class="version-selector">
```

Comment:
probably we should do an own custom element like `<ff-docu-version-select>` to avoid the code duplication of template and styling